lib/module.js

```diff
@@ -56,11 +56,11 @@ export default async function feed () {
 
 function resolveContentType (type) {
   const lookup = {
-    rss2: 'application/rss+xml',
-    atom1: 'application/atom+xml',
-    json1: 'application/json'
+    rss2: 'application/rss+xml;',
+    atom1: 'application/atom+xml;',
+    json1: 'application/json;'
   }
-  return lookup.hasOwnProperty(type) ? lookup[type] : 'application/xml'
+  return (lookup.hasOwnProperty(type) ? lookup[type] : 'application/xml;') + ' charset=UTF-8'
```

Comment:
Could you add a semicolon here instead of adding that to the lines 59-61 ☺️?

```suggestion
  return (lookup.hasOwnProperty(type) ? lookup[type] : 'application/xml') + '; charset=UTF-8'
```

Reply:
Sure, your code is more pretty